cms/models.py

```diff
@@ -327,3 +338,24 @@ class AlumniPage(BootcampRunChildPage):
 
     class Meta:
         verbose_name = "Alumni Section"
+
+
+class LearningResourcePage(BootcampRunChildPage):
+    """
+    Page that holds the learning resources for a product
+    """
+
+    heading = models.CharField(
+        max_length=100,
+        default="Learning Resources",
+        help_text="The heading to display on this section.",
+    )
+    items = StreamField(
+        [("item", LearningResourceBlock(max_num=3))],
```

Comment:
`max_num` seems me the part of `StreamBlock` rather than `StreamField`. 
By that CMS user seems me able to add more than 3 lists.

Reply:
It can either passed as keyword argument to the block as its being done here or by the Meta class attribute. It beings passed in the `StreamBlock` rather than the `StreamField`.

Reply:
The issue is that I'm using with the `StructBlock`, need to replace it with `StreamBlock`.

Reply:
I don't want to use `StreamBlock` that changes the cms model design too so I am removing the max limit as there is no hard rule for that.